lib/Tooling/Refactor/RefactoringContinuations.h

```diff
@@ -154,13 +165,46 @@ class PersistentToASTSpecificStateConverter {
     return Results;
   }
 
+  template <typename T>
+  bool addConvertible(
```

Comment:
No, since they're only produced by queries and aren't state. Therefore we just need to convert them back to `Decl *`.